lib/puppet/transaction.rb

```diff
@@ -322,7 +322,7 @@ def prefetch(provider_class, resources)
     begin
       provider_class.prefetch(resources)
     rescue LoadError, Puppet::MissingCommand => detail
-      Puppet.log_exception(detail, "Could not prefetch #{type_name} provider '#{provider_class.name}': #{detail}")
+      Puppet.log_exception(detail, _("Could not prefetch #{type_name} provider '#{provider_class.name}': #{detail}"))
```

Comment:
A comment would be good, I think. In your opinion, does this message need to be translated at all?